packages/showcases/react/stories/components/VtmnLink/VtmnLink.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { VtmnLink } from '@vtmn/react';
+
+import { Meta, Story } from '@storybook/react';
+
+export default {
+  title: 'Components/VtmnLink',
+  component: VtmnLink,
+  argTypes: {
+    href: {
+      type: { name: 'string', required: true },
+      description: 'The href of the link.',
+      defaultValue: '#',
+      control: { type: 'text' },
+    },
+    target: {
+      type: { name: 'string', required: true },
+      description: 'The target of the link.',
+      defaultValue: '_blank',
+      control: { type: 'text' },
+    },
```

Comment:
Target mustn't be required. Indeed, when we need to have an internal link, in HTML we don't specify it.
```html
<a href="/page-2.html">Go to next page</a>
```